src/common/decorators/swagger.ts

```diff
@@ -126,11 +126,15 @@ type SupportedHttpStatusCodes =
  * @param messages - key value map of HTTP response code to their description that will be passed to appropriate `MethodDecorator`
  */
 export function ApiTransactionFailedResponse(
-  messages: Partial<Record<SupportedHttpStatusCodes, string>>
+  messages: Partial<Record<SupportedHttpStatusCodes, string | string[]>>
```

Comment:
```suggestion
  messages: Partial<Record<SupportedHttpStatusCodes, string[]>>
```
It's not a big deal adding in an extra `[]` for a single item and then it's easier to add a second.

Reply:
`Arrary.isArrary` -> `rawDescription.length > 1` ?

I won't block approval on this though, either way is OK